src/VolumeLoader.ts

```diff
@@ -28,6 +28,34 @@ interface PackedChannelsImage {
 }
 type PackedChannelsImageRequests = Record<string, HTMLImageElement>;
 
+// Preferred spatial units in OME-Zarr are specified as full names. We want just the symbol.
+// See https://ngff.openmicroscopy.org/latest/#axes-md
+function spatialUnitNameToSymbol(unitName: string): string | null {
+  const unitSymbols = {
+    "angstrom":   "Å",
+    "decameter":  "dam",
+    "foot":       "ft",
+    "inch":       "in",
+    "meter":      "m",
+    "micrometer": "μm",
+    "mile":       "mi",
+    "parsec":     "pc",
+    "yard":       "yd",
+  };
+  if (unitSymbols[unitName]) {
+    return unitSymbols[unitName];
+  }
+
+  // SI prefixes not in unitSymbols are abbreviated by first letter, capitalized if prefix ends with "a"
+  if (unitName.endsWith("meter")) {
+    const capitalize = unitName[unitName.length - 6] === "a";
+    const prefix = capitalize ? unitName[0].toUpperCase() : unitName[0];
+    return prefix + "m";
+  }
+
+  return null;
```

Comment:
just curious, might it simplify things to return "" instead?

Reply:
you know what, it would

Reply:
Nevermind - that's there in case an unrecognized unit is specified, so that the loader can fall back to the full name at the call site. It could still return `unitName` instead though.